config/locales/server.en.yml

```diff
@@ -1702,6 +1702,9 @@ en:
     title: "The name of this site. Visible to all visitors including anonymous users."
     site_description: "Describe this site in one sentence. Visible to all visitors including anonymous users."
     short_site_description: "Short description in a few words. Visible to all visitors including anonymous users."
+    extended_site_description: "EXPERIMENTAL: A longer description for your site that supports Markdown syntax. Visible to all visitors including anonymous users."
```

Comment:
No need to mark these experimental; I think just make them `hidden: true` and only make them editable from the about config area for now.

Reply:
I made the new settings hidden, but now I've realized that hidden settings cannot be retrieved via the `Admin::SiteSettingsController#index` endpoint which is called by the about config page and specs are failing because of that.

I'm not super sure about adding `include_hidden: true` here:

https://github.com/discourse/discourse/blob/4a6b79dead474c459875d6342b7ffd90b462716d/app/controllers/admin/site_settings_controller.rb#L14-L19

because there are probably some settings that are super secret that even admins should be able to see. What we should we do here?

Reply:
Ah I didn't see this comment, and I just sent a comment about `include_hidden`. Hmm it's quite tricky, and you are right we probably shouldn't make it too broad.

What do you think about this? We add a `ADMIN_CONFIG_AREA_ALLOWLISTED_HIDDEN_SETTINGS` const in `Admin::SiteSettingsController`, and then add another param to `all_settings` called `filter_allowed_hidden` that passes that array in? Then if `include_hidden` is `true` and `filter_allowed_hidden` is passed in, we filter out any hidden settings that aren't in `filter_allowed_hidden`

Kind of weird and a bit messy but I don't see a better way for now. Also...for these particular settings it's only temporary, once we finish all this about page work we can make these site settings visible.

Reply:
I've done this in https://github.com/discourse/discourse/pull/27492/commits/45730d0039c77fb7c76592cded852618b28fcdc4.